tests/ipv6-evasion/ipv6-dos-with-ext-headers-1/test.yaml

```diff
@@ -0,0 +1,10 @@
+requires:
+    features:
+        - HAVE_LIBJANSSON
+
+checks:
+    - filter:
+        count: 34114
```

Comment:
is there a reason for testing such a large number? I assume these are not all unique packets?

Reply:
well that's one of the things to discuss, the pcap tries to DoS by using those ext headers in a way that can increase decoding load. So one might think about a more advanced rule or decoder events.